cmd/replay/main.go

```diff
@@ -250,22 +250,20 @@ func main() {
 		// TODO test: https://github.com/EspressoSystems/espresso-sequencer/issues/772
 		if chainConfig.Espresso {
 			hotShotCommitment := getHotShotCommitment(seqNum)
-			jst := message.Message.Header.BlockJustification
+			txs, jst, err := arbos.ParseEspressoMsg(message.Message)
```

Comment:
In the replay wasm, we parse the the message and get the raw txes and jst to validate the hotshot commitment